wheelDia: 0.005035
wheelBase: 2400.0
Track: 2800.0
fakewheelDia: 0.1007
fakewheelBase: 0.240
fakeTrack: 0.280